PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -239,6 +238,17 @@ public void finished(boolean aborted, int errors, int warnings, CompileContext c
                             if (aborted || errors != 0) {
                                 showCompilerErrorMessage(compileContext);
                                 jsmOpt.ifPresent((jsm) -> jsm.setJobRunningState(false));
+
+                                postEventProperty.put("IsSubmitSucceed", "false");
+                                postEventProperty.put("SubmitFailedReason", StringUtils.join(
+                                        compileContext.getMessages(CompilerMessageCategory.ERROR),
```

Comment:
property values should have a limit length, or we will get exception here. may be less than 50 characters?